planner/core/task.go

```diff
@@ -1275,14 +1275,14 @@ func BuildFinalModeAggregation(
 				partialCursor++
 			}
 			if aggFunc.Name == ast.AggFuncAvg {
-				cntAgg := *aggFunc
+				cntAgg := aggFunc.Clone()
```

Comment:
Is shallow copy not enough here? Why should we change it to `Clone()`?

Reply:
If we use shallow-copy, cntAgg and sumAgg will share same arg. In "injectExtraProj", changing arg in sumAgg stands for changing arg in CntAgg.

Reply:
Could you please add a code comment here for explanation?